allennlp/data/fields/text_field.py

```diff
@@ -88,29 +94,32 @@ def get_padding_lengths(self) -> Dict[str, int]:
 
         # Each indexer can return a different sequence length, and for indexers that return
         # multiple arrays each can have a different length.  We'll keep track of them here.
-        for indexer_name, indexer in self._token_indexers.items():
+        for token_index, indexer in self._token_indexers.items():
```

Comment:
I don't understand why you changed this name, and I think it was an error.

Reply:
Yes, you are right.